__tests__/Indexer.test.js

```diff
@@ -20,10 +20,6 @@ describe('Indexer', () => {
   const indexer = new Indexer()
 
   describe('constructor()', () => {
-    it('creates a client with the configured endpoint URL', () => {
-      expect(indexer.client.transport._config.host).toEqual(config.get('indexUrl'))
```

Comment:
This is removed because it uses an internal API, doesn't add code coverage, and was of arguable value all along.